### PR TITLE
chore: add css variable component to storybook

### LINKED ISSUE #2513: "Component tokens" component
Make an internal component that automates documentation of a components "component tokens" to be used in Storybook (and Storefront in the future).

Notes:
- Use postcss plugins in js to parse our css
- Let `import` do the work and pass the content to the component
- List all CSS variables in a table
- Maybe support comments in code thats also interpreted?

Something along the lines of:
Input CSS:
```css

.ds-button {
  --dsc-button-background--active: var(--ds-color-accent-base-active); /** Button background */
  …
}
```
Component use:
```tsx
import css from '@digidr/designsystemet-css/button'

<ComponentTokens css={css} />

```
Rendered table:
|Token|Description|Value|
|-|-|-|
|`--dsc-button-background--active`| Button background | `var(--ds-color-accent-base-active)` |

### PR DESCRIPTION
**Pull request by Barsnes**
resolves #2513

also fixes misc things for components

Comments:
select no longer has a css file
textarea no longer has a css file

### CODE REVIEW

**Comment by Barsnes** [comment]
Adding this to review before I add it to all our docs

**Comment by mimarz** [comment]
Whats up with the Table styling? I doesn't looks like our table, seems SB styling is interfering.

**Comment by Barsnes** [comment]
> Whats up with the Table styling? I doesn't looks like our table, seems SB styling is interfering.

Right. this probably needs an `sb-unstyled` class

File: apps/storybook/docs-components/CssVariables/CssVariables.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import { Table } from '@digdir/designsystemet-react';
+import type React from 'react';
```
Comment (mimarz) [inline apps/storybook/docs-components/CssVariables/CssVariables.tsx:2]: Redundant
```suggestion
```

File: apps/storybook/docs-components/CssVariables/CssVariables.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import { useEffect, useState } from 'react';
+
+type CssVariablesProps = {
+  css: string;
+};
+
+export const CssVariables: React.FC<CssVariablesProps> = ({ css }) => {
```
Comment (mimarz) [inline apps/storybook/docs-components/CssVariables/CssVariables.tsx:9]: We haven't used `React.FC` anywhere else and using `React.` goes against our decided convention which is WIP [here](https://github.com/digdir/designsystemet/issues/2450).

Suggest setting this component up the same way we do with others. In this case, just a named function would do?

**Review by mimarz — CHANGES_REQUESTED** [review]

File: apps/storybook/docs-components/CssVariables/CssVariables.tsx (hunk 2 above)
Comment (Barsnes) [reply apps/storybook/docs-components/CssVariables/CssVariables.tsx]: Commited a change with a named function

Comment (mimarz) [reply apps/storybook/docs-components/CssVariables/CssVariables.tsx]: being, add `...rest`, I dont see a use for `forwardRef` on this component.

Comment (Barsnes) [reply apps/storybook/docs-components/CssVariables/CssVariables.tsx]: Might as well add it, we might want it later on when we move documentation to our storefront

**Comment by mimarz** [comment]
Looks good!

**Pull request merged**